ports/filament/vcpkg.json

```
@@ -0,0 +1,22 @@
+{
+  "name": "filament",
+  "version-string": "1.9.20",
```

```suggestion
  "version-semver": "1.9.20",
```
This field should be more proper. Please see https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.reference.md